lib/api-group.js

```diff
@@ -23,6 +28,10 @@ class ApiGroup {
     this.requestOptions.cert = options.cert;
     this.requestOptions.key = options.key;
 
+    if (options.hasOwnProperty('insecureSkipTlsVerify')) {
```

Comment:
why hasOwnProperty. if we want that we should also not use it off the object since people could override it